reputation/reputation.go

```diff
@@ -0,0 +1,476 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package reputation
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/google/uuid"
+	// import of sqlite3 for side effects
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// RepRow is the Data type for Rows in Reputation table
+type RepRow struct {
+	name               string
+	timestamp          string
+	uptime             int
+	auditSuccess       int
+	auditFail          int
+	latency            int
+	amountOfDataStored int
+	falseClaims        int
+	shardsModified     int
+}
+
+// startDB starts a sqlite3 database from the file path parameter
+func startDB(filePath string) *sql.DB {
+	db, err := sql.Open("sqlite3", filePath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return db
+}
+
+// createTable creates a table in sqlite3 based on the create table string parameter
+func createTable(createStmt string, db *sql.DB) {
+	_, err := db.Exec(createStmt)
+	if err != nil {
+		log.Printf("%q: %s\n", err, createStmt)
+		return
```

Comment:
also still want to return the error instead of logging. i'll stop commenting on that for this review and down, but wherever there's an error, the function should return it